pkg/plugins/runtime/k8s/controllers/gatewayapi/mesh_http_route_conversion.go

```diff
@@ -102,10 +102,25 @@ func (r *HTTPRouteReconciler) gapiToKumaMeshRule(
 	ctx context.Context, mesh string, route *gatewayapi.HTTPRoute, rule gatewayapi.HTTPRouteRule,
 ) (v1alpha1.Rule, error) {
 	var matches []v1alpha1.Match
+	var filters []v1alpha1.Filter
 	var backendRefs []v1alpha1.BackendRef
 
 	for _, gapiMatch := range rule.Matches {
-		matches = append(matches, r.gapiToKumaMeshMatch(gapiMatch))
+		match, ok := r.gapiToKumaMeshMatch(gapiMatch)
+		if !ok {
+			continue
+			// TODO set condition
+		}
+		matches = append(matches, match)
+	}
+
+	for _, gapiFilter := range rule.Filters {
+		filter, ok := r.gapiToKumaMeshFilter(ctx, mesh, route.Namespace, gapiFilter)
+		if !ok {
```

Comment:
What are the situations when `ok` is false? Is it the user's input issue? If so, should we replace `ok` with `err`?

Reply:
it's also already tracked in the issue